HW15/src/FileLoggerConfiguration.java

```diff
@@ -0,0 +1,83 @@
+import java.io.File;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.Date;
+
+public class FileLoggerConfiguration {
+
+    private static double maxFileSize = 0.3;
+    private static String filePath = "src/";
+    private static File file = createFile();
+    private LoggingLevel loggingLevel;
+    private LogStructure logStructure;
+
+    public FileLoggerConfiguration() {
```

Comment:
It looks like that someone can create a "bad" FileLoggerConfiguration object with missing fields like `loggingLevel` and `logStructure`.